tools/ci_build/github/azure-pipelines/templates/py-packaging-training-cuda-stage.yml

```diff
@@ -57,7 +57,7 @@ stages:
 
   variables:
     - name: isMain
-      value: ${{ eq(variables['Build.SourceBranch'], 'refs/heads/main') }}
+      value: ${{ or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/rel-')) }}
```

Comment:
Have you tested it?

Reply:
yes, tested it on another branch with my name in the branch: 

```
Branch is refs/heads/bmeswani/update_packaging_pipeline_2
isMain is True
final_storage is --final_storage
```
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=232337&view=logs&j=8b05ce18-8d08-54e4-f752-3fd61fa2b2b4&t=db046059-25e9-5300-9b86-4ae474705954&l=12